### PR TITLE
add an example for elasticsearch/kibana

### PR DESCRIPTION
**Pull request by nicks**
Hello @jazzdan, @landism,

Please review the following commits I made in branch nicks/elastic:

d4bc1616c8b2d54ce7e24491a9c01993c00a6995 (2020-07-07 14:06:05 -0400)
add an example for elasticsearch/kibana

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: kibana/Tiltfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+load('ext://helm_remote', 'helm_remote')
+load('ext://namespace', 'namespace_create')
+
+namespace_create('elastic-system')
+
+operator_yaml = local('curl -sSL https://download.elastic.co/downloads/eck/1.1.2/all-in-one.yaml')
```
Comment (jazzdan) [inline kibana/Tiltfile:6]: Any reason not to include this YAML in the repo and remove a network dependency?

File: kibana/Tiltfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+             objects=['quickstart:elasticsearch'],
+             extra_pod_selectors=[{'elasticsearch.k8s.elastic.co/cluster-name': 'quickstart'}],
+             resource_deps=['elastic-operator'])
+k8s_resource(new_name='kibana',
+             objects=['quickstart:kibana'],
+             extra_pod_selectors=[{'kibana.k8s.elastic.co/name': 'quickstart'}],
+             resource_deps=['elastic-operator', 'elasticsearch'],
+             port_forwards=5601)
+
+local_resource('kibana-password',
```
Comment (jazzdan) [inline kibana/Tiltfile:22]: Clever!

**Review by jazzdan — APPROVED** [review]

File: kibana/Tiltfile (hunk 1 above)
Comment (nicks) [reply kibana/Tiltfile:6]: mostly it's 100K of yaml and isn't pedagogically that interesting

(though it does slow down the facets UI :grimacing: )

File: kibana/Tiltfile (hunk 2 above)
Comment (nicks) [reply kibana/Tiltfile:22]: ya, I feel like a lot of dashboards you set up on kubernetes have a whole auto-generated password dance, and i wonder if we could help people make nicer interfaces around this (for another example, see: https://github.com/tilt-dev/tilt/issues/1851)

**Pull request merged**